### PR TITLE
Add a Frontend Project - Playable Piano #2310

### LINKED ISSUE #2310: Add a Frontend Project - Playable Piano
**Describe the project you want to add with tech stack**
This will be a basic Frontend Project with the help of HTML, CSS and JavaScript.

**Expected behavior**
The user can experience a real time piano experience with this project, by pressing the keys of piano different tunes will be played.

**Screenshots (optional)**
image

**Additional context (optional )**

### PR DESCRIPTION
**Pull request by Atulsingh30**
# Title and Issue number 

Add a Frontend Project - Playable Piano #2310

Close #2310 
 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@Atulsingh30 please remove the image from PR and upload it somewhere in cloud and use the link of the image to load the image into your Project.

**Comment by Atulsingh30** [comment]
I have changed the image, have a look. Tell me if any change is required @MohitGupta121

File: Frontend-Projects/Playable Piano/image.png
Comment (MohitGupta121) [inline Frontend-Projects/Playable Piano/image.png:1]: @Atulsingh30 no all images are still there `.wav`

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@

**Comment by Atulsingh30** [comment]
Please check now! @MohitGupta121

File: Frontend-Projects/Playable Piano/image.png
Comment (Atulsingh30) [reply Frontend-Projects/Playable Piano/image.png:1]: Check now once if its correct

**Pull request merged**